content/docs/12-enterprise-version/03-deploying-palette-with-helm.md

```diff
@@ -320,6 +296,51 @@ If you notice that the pods in the `hubble-system` namespace are not initializin
 
 </WarningBox>
 
+
+# Upgrade Palette
+
+
+
+To upgrade Palette with a new Helm release, use the following steps. <br /> <br />
+
+1. Download the new version of the Helm Chart.
+
+
+
+2. Extract the new **values.yaml** file from the Helm Chart with the following command:
+
+    <br />
+
+    ```shell
+    tar xzvf /path/to/chart.tgz spectro-mgmt-plane/values.yaml
+    ```
+
+
+3. Compare the new **values.yaml** against the original **values.yaml** you used for the initial Palette installation. Address any new parameters added to the values file.
+
+
+
+
+4. Issue the following command to upgrade Palette. Use the same **values.yaml** file you used for the Palette installation. 
+
+    <br />
+
+    ```shell
+    helm upgrade palette /path/to/chart.tgz --file /path/to/orginal_values.yaml
+    ```
+ 
+
+## Post-Install Configuration Values
+
+The values you specified in the **values.yaml** file all fall under the parameter section `values.config` and are stored in the `configserver-cm` ConfigMap. 
```

Comment:
Small Suggestion:
The values you specified in the **values.yaml** file all fall(falls) under the parameter section `values.config` and are(is) stored in the `configserver-cm` ConfigMap.